lib/sys-utils/tests/fd_mmap_memory.rs

```diff
@@ -0,0 +1,124 @@
+// // TODO: make this a sensible test before merge NOTE: commented out since it's mostly here for demonstration purposes.
+// use std::sync::Arc;
```

Comment:
Do we want to remove this file from the PR since is all commented out?

Reply:
Id rather keep it in so we are less likely to forget about fixing it up when we have time.

Reply:
#3422